### PR TITLE
Made image aspect checked by default

### PR DESCRIPTION
**Pull request by ankit-kumar-dwivedi**
Fixes: #771 
@morevnaproject please review this when you have time [^_^]

### CODE REVIEW

**Comment by morevnaproject** [comment]
Thanks! Reviewing now. ^__^

File: synfig-studio/src/gui/renddesc.cpp [hunk 1]
```diff
@@ -151,7 +154,7 @@ Widget_RendDesc::refresh()
 	entry_duration->set_value(rend_desc_.get_duration());
 
 	adjustment_fps->set_value(rend_desc_.get_frame_rate());
-
+	//zzzzzzzzzzzzzzzzzzzzz
```
Comment (morevnaproject) [inline synfig-studio/src/gui/renddesc.cpp]: I suggest to remove this line. :satisfied:

**Comment by morevnaproject** [comment]
It looks like there is some misunderstanding here. ^__^ As per my comment - https://github.com/synfig/synfig/issues/771#issuecomment-476298262 - we need "Image Aspect" checkbox enabled. This PR enables "Image Span" (which is already enabled by default).

File: synfig-studio/src/gui/renddesc.cpp (hunk 1 above)
Comment (ankit-kumar-dwivedi) [reply synfig-studio/src/gui/renddesc.cpp]: That's my way to mark things as important 😂

**Comment by ankit-kumar-dwivedi** [comment]
> It looks like there is some misunderstanding here. ^__^ As per my comment - [#771 (comment)](https://github.com/synfig/synfig/issues/771#issuecomment-476298262) - we need "Image Aspect" checkbox enabled. This PR enables "Image Span" (which is already enabled by default).

Will fix this when I get home :-)

**Comment by morevnaproject** [comment]
Also, I think that ```renddesc.cpp``` is not the best place for such modifications. I suggest to take a look at the code which responsible for creating new documents (in ```app.cpp```) - this is where it sets all parameters of renddesc for new files.

**Comment by ankit-kumar-dwivedi** [comment]
> Also, I think that `renddesc.cpp` is not the best place for such modifications. I suggest to take a look at the code which responsible for creating new documents (in `app.cpp`) - this is where it sets all parameters of renddesc for new files.

Okay @morevnaproject I'll keep it on my mind! ^_^

**Comment by ankit-kumar-dwivedi** [comment]
@morevnaproject done changes please have a look :smile:

**Comment by morevnaproject** [comment]
Code now looks good! But, I have tested the changes and the resize didn't worked for me. From this comment - https://github.com/synfig/synfig/issues/771#issuecomment-473236624 - I figured out that we also need to disable "Pixel Aspect" checkbox (it is enabled by default).

**Comment by ankit-kumar-dwivedi** [comment]
@morevnaproject disabled pixel aspect by default!

**Comment by morevnaproject** [comment]
Looks god now. Thank you! ^__^

**Pull request merged**